packages/ember/lib/index.js

```diff
@@ -200,8 +200,10 @@ Object.defineProperty(Ember, 'LOG_BINDINGS', {
   and reporting code.
```

Comment:
line 390 refers to `Ember.String.loc`.  Those types of refernces are what we are looking to update to `String.loc`